py/jupyterlite/src/jupyterlite/constants.py

```diff
@@ -76,9 +76,15 @@
 #: this is arrived at by inspection
 NPM_SOURCE_DATE_EPOCH = 499162500
 
-#: the only kind of wheel piplite understands
+#: the only kind of noarch wheel piplite understands
 NOARCH_WHL = "py3-none-any.whl"
 
+#: the only kind of binary wheel piplite understands
+WASM_WHL = "emscripten_wasm32.whl"
```

Comment:
@richardotis if you had the chance, some eyes on this with your custom wheels  would be informative... we likely won't spin up a pyodide build environment in this repo at this time, but having some idea of whether stuff is working would be excellent.